featuretools/tests/io_tests/test_from_sql.py

```diff
@@ -0,0 +1,39 @@
+# flake8: noqa: F811
```

Comment:
It looks like using fixtures in this way causes a linting error, and I see some other people on SO with the same issue :/

Reply:
Instead of defining the fixtures in `testing_utils` and importing them we could create a `featuretools/tests/io_tests/conftest.py` file and define the fixtures there. From the looks of [this documentation](https://docs.pytest.org/en/latest/example/simple.html#package-directory-level-fixtures-setups) we could then use the fixtures in this file without needing to import them.

Reply:
Awesome - that worked, updated.